src/data/timer-first-interaction.js

```diff
@@ -0,0 +1,8 @@
+const listener = document.addEventListener('pulse-send-pagemonitor', evt => {
```

Comment:
Yeah, that's exactly where I stand. Might go through and webpack them once they're all finished, but just trying to be practical right now.